### PR TITLE
first attempt at writing style overhaul

### PR DESCRIPTION
**Pull request by hollsk**
Input requested :-)

I've tried to keep it concise, but there may be things missed out. Perhaps there are specific guidelines for _technical_ writing I don't know of? @typothecary ?

### CODE REVIEW

File: contributing/communications.md [hunk 1]
```diff
@@ -17,27 +17,38 @@ Hello, so you're interested in helping out with our communications? We're pretty
...
 
-### Write Like a Human...
 
-Be professional, yes, but also conversational. We don't want to sound like robots, and should certainly never be communicating on behalf of the Pa11y project in unfeeling business-speak. At all times our users should feel like they're talking to a regular human.
-
-### ...But a Nice Human
+### Tone of Voice Guidelines
 
+- Write like a human  
+Be professional, yes, but also conversational. We don't want to sound like robots, and should certainly never be communicating on behalf of the Pa11y project in unfeeling business-speak. At all times our users should feel like they're talking to a regular human.
```
Comment (rowanmanning) [inline contributing/communications.md]: I notice you're using single line breaks here. I'm afraid they don't work :( oddly they render locally but when you push to GitHub pages they don't.

I've been using headings in lists for this, not sure whether it's correct or not but it works. [E.g. here](https://github.com/pa11y/pa11y.github.io/commit/71bad37160496cfb0965db7cc033d58c0f9a6797#diff-414a1a0a980a16f36003309fa914c17dR44)

File: contributing/communications.md [hunk 2]
```diff
@@ -17,27 +17,38 @@ Hello, so you're interested in helping out with our communications? We're pretty
...
-### Even in Documentation
+Follow Plain English guidelines to help safeguard the accessibility of your written content. It'd be ironic for an a11y project to produce documents that are hard to understand! All relevant guidelines for [writing well][writing-well] apply. In particular:
 
-We're not saying that the documentation should read like a conversation, that would probably be really jarring and unhelpful when describing technical things. However introductions and any signposting should be friendly and helpful.
+- Be concise  
+  - Check any sentences with more than 25 words to see if you can split them to make them clearer. 
+  - Use contractions (e.g. "can't", "it's").
+  - Be brisk but not terse.
+- Be specific
+  - Use the active voice instead of the passive voice (e.g. "NPM will install the dependencies" instead of "the dependencies will be installed").
```
Comment (rowanmanning) [inline contributing/communications.md]: Maybe we could link to [Hemingway](http://www.hemingwayapp.com/) for help on this?

**Comment by rowanmanning** [comment]
Looking great! Do we need to mention avoiding metaphor/simile/sayings?

File: contributing/communications.md (hunk 1 above)
Comment (hollsk) [inline contributing/communications.md]: Good shout, I'll copy your style, thanks!

**Comment by hollsk** [comment]
I added Hemingway and a line about similes/metaphors/slang, but why don't my headings work? 😱

**Comment by rowanmanning** [comment]
👍 happy to merge unless you're hoping for some feedback from @typothecary?

**Comment by hollsk** [comment]
The more the merrier, I reckon, but she might be busy! We could give it until the end of the day and merge if no comments?

**Pull request merged**